<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Touch</class>
 <widget class="QWidget" name="Touch">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Touch</string>
  </property>
  <property name="toolTip">
   <string>Touch</string>
  </property>
  <property name="statusTip">
   <string>Touch</string>
  </property>
  <property name="whatsThis">
   <string>Touch</string>
  </property>
  <widget class="QTabWidget" name="touchTab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>480</width>
     <height>450</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="panelTab">
    <attribute name="title">
     <string>Panel</string>
    </attribute>
    <widget class="QTabWidget" name="panelResolution">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>476</width>
       <height>419</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="narrowTab">
      <property name="toolTip">
       <string/>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <attribute name="title">
       <string>Narrow (800*480)</string>
      </attribute>
      <widget class="QFrame" name="narrowArea">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>73</y>
         <width>400</width>
         <height>240</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="wideTab">
      <property name="toolTip">
       <string/>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <attribute name="title">
       <string> Wide (1280*480)</string>
      </attribute>
      <widget class="QFrame" name="wideArea">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>118</y>
         <width>400</width>
         <height>150</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="padTab">
    <attribute name="title">
     <string>Pad</string>
    </attribute>
    <widget class="QTabWidget" name="padCoord">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>476</width>
       <height>419</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="TouchPadWidget" name="relativeTab">
      <attribute name="title">
       <string>Relative</string>
      </attribute>
     </widget>
     <widget class="TouchPadWidget" name="absolutePad">
      <attribute name="title">
       <string>Absolute</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="touchTip">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>475</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TouchPadWidget</class>
   <extends>QWidget</extends>
   <header location="global">touchpadwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
